neon_homeassistant_skill/__init__.py

```diff
@@ -50,6 +50,35 @@ def initialize(self):
             self.handle_set_light_color_response,
         )
 
+        resp = self.bus.wait_for_response(Message("ovos.phal.plugin.homeassistant.check_connected"))
+        if resp and resp.data.get("connected"):
+            self.log.debug("PHAL plugin connected to HA")
+            return
+        if not resp:
+            self.log.error("Home Assistant PHAL plugin not installed or not running!")
+            self.disable_unused_intents()
+            return
+
+        self.log.info("PHAL Plugin not connected to HomeAssistant")
+        self.disable_unused_intents()
+
+    def disable_unused_intents(self):
+        for intent in (
```

Comment:
Just adding some notes for testing/troubleshooting.
`self.intent_names` is a list of intent names without `skill_id` prefix (regardless of being registered or deactivated)
`self.detached_intents` is a list of tuple `(intent_name, intent_obj)` for detached/disabled intents
`self.registered_intents` is a list of tuple  `(intent_name, intent_obj)` for non-detached intents